assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -520,7 +529,23 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
     const filterVars = createCapiObjectFromStateVars(simLife.currentState, simLife.domain);
     if (filterVars && Object.keys(filterVars)?.length !== 0) {
       handleIFrameSpecificProperties(simLife.currentState, simLife.domain);
-      sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.VALUE_CHANGE, filterVars);
+      writeCapiLog('SENDING SIM CONFIG DATA !!!!', 3, {
+        simLife,
+        configData: filterVars,
+      });
+      Object.keys(filterVars).forEach((variable: any) => {
+        const formatted: Record<string, unknown> = {};
+        const simVariable = filterVars[variable];
```

Comment:
I found this a bit confusing since you are mutating items via reference.  I think that you might as well just use it directly; so like
```
formatted[variable] = filterVars[variable];
if (typeof formatted[variable].value === 'string') {
   formatted[variable].value = templatizeText(formatted[variable].value, simLife.snapshot, scriptEnv);
}
sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.VALUE_CHANGE, formatted);
```